guides/README.md

```diff
@@ -46,7 +46,7 @@ Variables cannot be used in shell or code examples. To use them, use "attributes
 
 Hide or show specific blocks, paragraphs, warnings or chapters via special variable called "build". Its value can be set either to "foreman" or "satellite":
 
-	ifeval::["{build}" == "upstream"]
+	ifeval::["{build}" == "foreman"]
```

Comment:
This was a typo, the variable is really called "foreman" and not "upstream".

Reply:
Hi Lzap, 

This works for me. 
I have made several changes to match the asciidoc standards that we are hoping to use throughout the guide. Sometimes we haven't adhered to this in the live guides, but are slowly moving towards full compliance with Asciidoc conventions. You don't need to worry about this because I will always fix it. I rewrote some of the sentences to correct the grammar etc. If I have misunderstood something, please let me know. 
I go on PTO at lunchtime, so if you want, you can merge this. I have no problem with the content. If it isn't urgent, I can do it on my return next Wednesday. Thanks for guiding me through this.

Reply:
Thanks, looks good.